apps/web/src/cdp/api/cb-gpt.ts

```diff
@@ -39,7 +39,8 @@ export async function queryCbGpt(query: CbGptQuery): Promise<QueryCbGptResponse>
     }
 
     if (response.status === 401 || response.status === 403) {
-      throw new Error('Unauthorized access: ensure the calling IP is permitted.');
+      console.error(errorResponse);
```

Comment:
Are you sure you want this?

Reply:
yeah at least for now, this will allow us to figure out what is going wrong with the sb-gpt setup. The error I'll use to try to track if we are getting this i any of our logging systems.